### ISSUE TITLE
Stetho dependency

### ISSUE THREAD

**Issue by tir38** [helpscout/beacon-android-sdk-sample#64]
Why does Helpscout jar have runtime dependency on Facebook's Stetho?

Just noticed this

```
+--- com.helpscout:beacon-core:1.0.6
|    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.3.31 -> 1.3.50 (*)
|    +--- org.jetbrains.kotlinx:kotlinx-coroutines-core:1.0.0 -> 1.3.1
|    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.3.50 (*)
|    |    \--- org.jetbrains.kotlin:kotlin-stdlib-common:1.3.50
|    +--- org.jetbrains.kotlinx:kotlinx-coroutines-android:1.0.0 -> 1.3.1
|    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.3.50 (*)
|    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.1 (*)
|    +--- com.squareup.okhttp3:okhttp:3.9.0 -> 4.2.2 (*)
|    +--- com.squareup.retrofit2:retrofit:2.3.0 -> 2.6.2 (*)
|    +--- com.squareup.retrofit2:converter-moshi:2.3.0
|    |    +--- com.squareup.retrofit2:retrofit:2.3.0 -> 2.6.2 (*)
|    |    \--- com.squareup.moshi:moshi:1.4.0
|    |         \--- com.squareup.okio:okio:1.11.0 -> 2.4.0 (*)
|    +--- ru.gildor.coroutines:kotlin-coroutines-retrofit:1.0.0
|    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.3.11 -> 1.3.50 (*)
|    |    +--- org.jetbrains.kotlinx:kotlinx-coroutines-core:1.0.1 -> 1.3.1 (*)
|    |    \--- com.squareup.retrofit2:retrofit:2.4.0 -> 2.6.2 (*)
|    +--- com.squareup.okhttp3:logging-interceptor:3.9.0 -> 4.2.2 (*)
|    +--- com.facebook.stetho:stetho:1.5.0 // <---------- HERE
|    |    +--- commons-cli:commons-cli:1.2
|    |    \--- com.google.code.findbugs:jsr305:2.0.1 -> 3.0.1
|    +--- com.facebook.stetho:stetho-okhttp3:1.5.0
|    |    +--- com.facebook.stetho:stetho:1.5.0 (*)
|    |    +--- com.google.code.findbugs:jsr305:2.0.1 -> 3.0.1
|    |    \--- com.squareup.okhttp3:okhttp:3.4.2 -> 4.2.2 (*)
|    +--- com.jakewharton.timber:timber:4.7.1 (*)
|    +--- com.jakewharton.threetenabp:threetenabp:1.1.1
|    |    \--- org.threeten:threetenbp:1.3.7
|    +--- org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.3.31 -> 1.3.50 (*)
|    \--- androidx.core:core:1.0.0 -> 1.1.0 (*)
```

**Comment by scottyab** [comment]
Hey @tir38 We use stetho for debugging but it wasn't mean to still be in the release. I'll look at tweaking the dependency settings to exclude.

**Comment by theBradfo** [comment]
Came here to post this same question!  @scottyab do you expect a release removing this soon, or is it safe to exclude as a transitive dependency in gradle?

**Comment by scottyab** [comment]
@theBradfo @tir38 yes, the PR has already been merged and will be in the next public release. Thanks for flagging with us.

**Comment by adrianne-helpscout** [comment]
https://helpscout.atlassian.net/browse/ANDSDK-213

**Comment by scottyab** [comment]
This has been removed and resolved in 1.0.9. Please reopen if you're still seeing the Stetho dependency. Thanks
